src/opensprinkler-card.ts

```diff
@@ -209,6 +209,11 @@ export class OpensprinklerCard extends LitElement {
   private _secondaryText() {
     const entities: EntitiesFunc = p => this._matchingEntities(p)
 
+    if (hasRainDelayActive(entities)) {
+      const stop_time = entities(isRainDelayStopTime).find(_ => true)?.state;
```

Comment:
Likewise this feels like a hacky approach to selecting the first (and only expected) item in the list, without failing if the list is empty due to renames, etc. Good enough? Other ideas?

Reply:
I think `entities(isRainDelayStopTime)[0]?.state` would have the same effect. If there are no rain delay entities `entities(isRainDelayStopTime)[0]` will be `undefined`.